python/foglamp/common/configuration_manager.py

```diff
@@ -298,10 +302,11 @@ def get_entry_val(k):
             else:
                 new_category_val = category_val
 
+            display_name = category_name if display_name is None else display_name
             audit = AuditLogger(self._storage)
             await audit.information('CONAD', {'name': category_name, 'category': new_category_val})
             payload = PayloadBuilder().INSERT(key=category_name, description=category_description,
-                                              value=new_category_val).payload()
+                                              value=new_category_val, display_name=display_name).payload()
             result = await self._storage.insert_into_tbl("configuration", payload)
             response = result['response']
             self._cacheManager.update(category_name, new_category_val)
```

Comment:
I believe, We need to cache this display name too!

Reply:
Yes, we should probably keep the display name in the cache